workflow/serializers.py

```diff
@@ -124,16 +163,12 @@ class Meta:
 
 class Parameter(ModelSerializer):
 
-    #included_serializers = {
```

Comment:
remove or add a comment explaining why it's here?

Reply:
You mean the model itself?

Reply:
oh whoops, that was my mistake. thought you were adding this commented code, not deleting it. you're all good :+1: